src/webportal/src/app/env.js.template

```diff
@@ -1,4 +1,5 @@
 window.ENV = {
+  restServerLauncherType: '${REST_SERVER_LAUNCHER_TYPE}',
```

Comment:
I have added a environment variable `launcherType` and it is already checked in. Please used that instead.